tests/Page/TemplateManagerTest.php

```diff
@@ -136,6 +144,51 @@ public function testRenderResponseWithDefaultParameters(): void
         );
     }
 
+    public function testTemplateShowingBreadcrumbIntoThePage(): void
+    {
+        $kernel = self::bootKernel();
+        // TODO: Simplify this when dropping support for Symfony 4.
+        // @phpstan-ignore-next-line
+        $container = method_exists($this, 'getContainer') ? self::getContainer() : $kernel->getContainer();
+
+        $requestStack = new RequestStack();
+        $requestStack->push(new Request());
+        $container->set('request_stack', $requestStack);
+
+        // Mocking snapshot
+        $pageMock = $this->createMock(PageInterface::class);
+        $pageMock->method('getName')->willReturn('Foo');
+        $pageMock->method('getParents')->willReturn([]);
+        $pageMock->method('getUrl')->willReturn('/');
+
+        // Mock Snapshot manager
+        $snapshotManagerMock = $this->createMock(SnapshotManagerInterface::class);
+        $transformerMock = $this->createMock(TransformerInterface::class);
+        $cmsSnapshotManagerMock = new CmsSnapshotManager($snapshotManagerMock, $transformerMock);
```

Comment:
I recommend to not name variables with mock suffix , this one is a mistake for example, it is not a mock

Reply:
true :/